build.gradle

```diff
@@ -59,4 +65,95 @@ subprojects {
   }
 
   apply from: "${rootDir}/gradle/dependencies.gradle"
+
+  plugins.withType(MavenPublishPlugin).withType(SigningPlugin) {
+    apply plugin: "org.shipkit.travis"
+    apply plugin: "org.shipkit.github-pom-contributors"
+    apply plugin: "org.shipkit.release-notes"
+    apply plugin: "org.shipkit.compare-publications"
+    publishing {
+      publications {
+        tasks.downloadPreviousReleaseArtifacts.previousSourcesJarUrl =
+            "https://repo1.maven.org/maven2/com/linkedin/coral/" + subproject.name + "/" + tasks.fetchReleaseNotes.previousVersion +
+            subproject.name + "-" + tasks.fetchReleaseNotes.previousVersion + "-sources.jar"
+        maven(MavenPublication) { publication ->
+          tasks.performRelease.dependsOn(tasks.publish)
+          from components.java
+          artifact(javadocJar) {
+            classifier = 'javadoc'
+          }
+          artifact(sourcesJar) {
+            classifier = 'sources'
+          }
+          pom {
+            groupId = 'com.linkedin.coral'
+            artifactId = subproject.name
+            name = 'Coral'
+            description = 'Coral is a SQL analysis, translation, and rewrite engine'
```

Comment:
Up to you, but you might consider having the name and description describe what the actual artifact is. As a consumer, I find that to be a nicer use of name + description here.